.github/workflows/ci.yml

```
@@ -272,7 +272,7 @@ jobs:
   # E2E 시나리오 테스트
   test-e2e-scenarios:
     needs: changes
-    if: ${{ needs.changes.outputs.supabase == 'true' }}
+    if: false
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)